docs/source/projects/architecture/content-architecture.rst

```diff
@@ -175,4 +188,4 @@ Below is a high level visual overview of project relationships. It is current as
 
 .. image:: /_static/_images/repos_map.png
    :width: 75%
-   :alt: Architecture diagram of project relationships
+   :alt: Architecture diagram of Jupyter project relationships from servers, applications, API, and kernels.
```

Comment:
Good lord. How do we make such a dense relationship map accessible? I feel like this is getting into new territory for me

Reply:
I personally think the best option would be to have this be a part of the main text with an empty alt attribute, but I know I do not have the time to write it at the moment. My current guess is that it would make sense to start by describing the overlapping content first to minimize repetition, then call out areas by section. 

Maybe it's the wrong attitude, but I was going with "something is better than nothing" for this. If it's a blocker, let me know and I'll find a way to get it done.

If we think the image is the way to go, I think this one needs a linked long description. Even that will be tricky with all the relationships!

Reply:
Agreed. This will need to be revisited in the future.